### PR TITLE
Update docusaurus.config.js

### PR DESCRIPTION
**Pull request by Pandademic**
Changed the config to add the n certain places and fix  some grammar

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -6,15 +6,15 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
...
+  tagline: 'The Universal programming language',
   url: 'https://unv.vercel.app',
   baseUrl: '/',
   onBrokenLinks: 'warn',
   onBrokenMarkdownLinks: 'warn',
   favicon: 'img/favicon.ico',
-  organizationName: 'unvlabs',
-  projectName: 'site',
+  organizationName: 'Unvlabs',
+  projectName: 'The UNV site',
```
Comment (ksenginew) [inline docusaurus.config.js]:
```suggestion
  projectName: 'site',
```
This should match the name of Github repo contains the site.
```
https://github.com/UnvLabs/site
site is taken from this url
```

**Comment by Pandademic** [comment]
Ready to merge , I believe?

**Comment by ksenginew** [comment]
Thanks.

> For documentation and other docs related changes, please do many changes to one branch and open a pull request. Otherwise we'll have issue number #10000 quickly.

**Comment by Pandademic** [comment]
Ok

**Pull request merged**